controllers/admin/AdminImagesController.php

```diff
@@ -427,12 +471,14 @@ public function postProcess()
                 } elseif ((int) Tools::getValue('PS_WEBP_QUALITY') < 0
                     || (int) Tools::getValue('PS_WEBP_QUALITY') > 100) {
                     $this->errors[] = $this->trans('Incorrect value for the selected WebP image compression.', [], 'Admin.Design.Notification');
-                } elseif (!Configuration::updateValue('PS_IMAGE_QUALITY', Tools::getValue('PS_IMAGE_QUALITY'))
+                }
+                /*elseif (!Configuration::updateValue('PS_IMAGE_QUALITY', Tools::getValue('PS_IMAGE_QUALITY'))
```

Comment:
Seems like you are removing this safety check that ensures that the quality cannot be 0? Why remove this check, this changes the initial controller validation process but I don't see why it's needed